website/docs/docs/cloud/manage-access/set-up-bigquery-wif.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Set up BigQuery WIF"
+description: "Learn how configure dbt Cloud and BigQuery to use workload identity federation"
+id: "set-up-bigquery-wif"
+pagination_next: null
+---
+
+# Set up BigQuery workload identity federation <Lifecycle status='beta,managed'/> 
+
+Workload identity federation (WIF) allows application workloads, running externally to dbt Cloud, to act as a service account without the need to manage service accounts or other keys. The following instructions will enable you to authenticate your BigQuery connection in dbt Cloud using WIF. 
+
+## Set up dbt Cloud
+
+To configure a BigQuery connection to use WIF authentication in dbt Cloud, you must set up a custom OAuth integration configured with details from the Entra application used as your workpool provider in GCP.
+
+In dbt Cloud: 
+
+1. Navigate to **Account settings** --> **Integrations** 
+2. Scroll down to the section for **Custom OAuth Integrations** and create a new integration, 
+3. Fill out all fields with the appropriate information from your Entra ID environment.
+    - The Application ID URI should be set to the expected audience claim on tokens issued from the Entra application. It will be the same URI your workpool provider has been configured to expect.
+    - You do not have to add the Redirect URI to your Entra application
+
+## Create connections in dbt Cloud
+
+To get started, create a new connection in dbt Cloud:
+
+1. Navigate to **Account settings** --> **Connections**.
+2. Click **New connection** and select **BigQuery** as the connection type.
+3. You will be presented with two schema options for the connection (both use the same adapter):
+    - **BigQuery:** Select this option to configure WIF.
+    - **BigQuery (Legacy):** Still valid for other connection types but does not support WIF.
```

Comment:
I know we had conversations back and forth on this but I think the new one supports all of the auth methods so we should just tell folks to use that one?